website/project/model.py

```diff
@@ -880,6 +875,11 @@ def pk(self):
         return self._id
 
     @property
+    def logs(self):
+        """ List of logs associated with this node"""
+        return NodeLog.find(Q('node', 'eq', self._id)).sort('date')
```

Comment:
Logs was formerly a field on the Node model, and it was an array. When `node.add_log` was run, it appended the new log onto the end of the logs list.  So oldest logs first, newest logs last.  I was matching previous behavior.